tools/ViralConsensus/compile.sh

```diff
@@ -1,6 +1,8 @@
+sed -i 's/unsigned char \*tmp = realloc(b->data, len);/unsigned char \*tmp = static_cast<unsigned char \*>(realloc(b->data, len));/g' htslib/cram/cram_io.h
```

Comment:
Thanks! Can you add a comment on top briefly mentioning why that change is needed?

Reply:
Done!